### PR TITLE
Create author

### PR DESCRIPTION
**Pull request by MenBui**
==> Title: create_author

==> Redmine: https://edu-redmine.sun-asterisk.vn/issues/19311

==> Attachment File:
Screenshot from 2019-12-20 09-16-40

### CODE REVIEW

**Comment by MenBui** [comment]
ready

**Comment by MenBui** [comment]
ready

File: app/models/book.rb [hunk 1]
```diff
@@ -5,10 +5,11 @@ class Book < ApplicationRecord
   belongs_to :author
   belongs_to :category
   belongs_to :publisher
-  has_many :borrowed_details
+  has_many :borrowed_details, dependent: :restrict_with_error
   has_many :comments, dependent: :destroy
   has_many :rates, dependent: :destroy
-  has_many :borroweds, through: :borrowed_details, dependent: :restrict_with_error
+  has_many :borroweds, through: :borrowed_details,
+  dependent: :restrict_with_error
```
Comment (manhbnt-0960) [inline app/models/book.rb]: indent

File: app/views/admin/authors/index.html.erb [hunk 2]
```diff
@@ -1,15 +1,22 @@
 <% provide :title, t("author_manage") %>
 <h2 class = "title"><%= t "author_manage" %></h2>
+<span>
+  <%= link_to new_admin_author_path, class: "btn btn-success" do %>
+  <%= t "create_author" %>
+  <i class="fa fa-plus"></i>
```
Comment (manhbnt-0960) [inline app/views/admin/authors/index.html.erb]: indent

**Comment by MenBui** [comment]
ready

**Pull request merged**